app/data_model/questionnaire_store.py

```diff
@@ -35,14 +33,14 @@ def save(self):
         logger.debug("Saving user data %s for user id %s", self.data, self.user_id)
         self.storage.store(data=self.data, user_id=self.user_id, user_ik=self.user_ik)
 
-    def encode_metadata(self, metadata):
-        self.data["METADATA"] = jsonpickle.encode(metadata)
-
-    def decode_metadata(self):
+    @property
+    def metadata(self):
         if "METADATA" in self.data:
```

Comment:
PEP8 has explicit return values, even if None: "explicit return statement should be present at the end of the function (if reachable)."; up to you and team to agree if this is something you want to stick to or not.